working/templates/automation-project/.template_config/template.json

```diff
@@ -0,0 +1,209 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Skyline Communications",
+  "classifications": [
+    "DataMiner"
+  ],
+  "tags": {
+    "language": "C#",
+    "type": "project"
+  },
+  "identity": "Skyline.DataMiner.AutomationScript.Project.Template",
+  "name": "DataMiner Automation Script Project",
+  "shortName": "dataminer-automation-project",
+  "sourceName": "$SCRIPTNAME$",
+  "symbols": {
+    "Author": {
+      "type": "parameter",
+      "datatype": "string",
+      "defaultValue": "",
+      "replaces": "$AUTHOR$",
+      "description": "The author."
+    },
+    "CreateDataMinerPackage": {
+      "type": "parameter",
+      "datatype": "bool",
+      "description": "Enables this project to generate a .dmapp and optionally publish to the online catalog as a stand-alone item. For multi-artifact packages please leave this empty and use an additional DataMiner Package Project.",
+      "defaultValue": "false",
+      "replaces": "$CREATEPACKAGE$"
+    },
+    "IncludeGitHubWorkflow": {
+      "type": "parameter",
+      "datatype": "choice",
+      "description": "Add/Overwrite a GitHub Workflow: Basic (build, test, publish) or Complete (Skyline Quality Gate). Warning: When adding a new Project to an existing solution or for dotnet-cli this will always be one folder up from your OUTPUT_DIRECTORY.",
```

Comment:
This isn't very clear. What do you mean with "or for dotnet-cli"?

Reply:
In command line. (in search if you open a cmd.exe, commandline, terminal or powershell) You can type  dotnet new dataminer-automation-project.  This will also create the Visual Studio Project you want. That's called the dotnet-cli.